src/execution/collectFields.ts

```diff
@@ -173,7 +174,7 @@ function collectFieldsImpl(
         break;
       }
       case Kind.FRAGMENT_SPREAD: {
-        const fragName = selection.name.value;
+        const fragmentName = selection.name.value;
```

Comment:
This was changed from the original, before we used to generate a key for each FragmentSpread to see whether it was the same as others in the selection-set however this isn't needed as discussed in https://github.com/graphql/graphql-js/pull/3835#discussion_r1101825832